compiler/compile-code/compile-code-piggy_extractor

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env jq-f
+# compile-code-plpy_extractor -- TODO Compiles plpy_extractors in the normalized JSON into code executable by the runner
+##
+.deepdive_ as $deepdive
+
+| $deepdive.execution.processes | to_entries[]
+| .value.name = .key | .value
+| select(.style == "plpy_extractor")
```

Comment:
It seems that the content in `compiler/compile-code/compile-code-piggy_extractor` and `compiler/compile-code/compile-code-plpy_extractor` should switch?